### ISSUE TITLE
Replace all `data.http.myip.body` by `data.http.myip.response_body` in networking.tf

### ISSUE THREAD

**Issue by tungbq** [tungbq/aws-lab-with-terraform#47]
**Background:**
- We got following warning in networking.tf:
```
│ Warning: Deprecated attribute
│
│   on networking.tf line 219, in resource "aws_security_group" "internet_facing_lb_sg":
│  219:     cidr_blocks      = ["${chomp(data.http.myip.body)}/32"]
│
│ The attribute "body" is deprecated. Refer to the provider documentation for details.
│
│ (and 3 more similar warnings elsewhere)
```
- For example:
https://github.com/tungbq/aws-lab-with-terraform/blob/7eebd5240ce8281c3e3817dfe451b013670aeac0/workshop/aws-3-tier-web/networking.tf#L275
https://github.com/tungbq/aws-lab-with-terraform/blob/7eebd5240ce8281c3e3817dfe451b013670aeac0/workshop/aws-3-tier-web/networking.tf#L219

**Solution:**
- Replace all `data.http.myip.body` by `data.http.myip.response_body` in networking.tf
- Document: https://registry.terraform.io/providers/hashicorp/http/latest/docs/data-sources/http#read-only
- Similar issue: https://github.com/jpetazzo/ampernetacle/issues/41

**Comment by Harshal662** [comment]
i can do that

**Comment by tungbq** [comment]
> i can do that

Much appreciated @Harshal662 for looking into this, although @vedangit has provided the fix already in #52 
Feel free to pick up other issues in this repo. Thanks
